bundles/framework/publisher2/resources/locale/fi.js

```diff
@@ -280,6 +281,13 @@ Oskari.registerLocalization(
                 "publishable": "Näytä vain julkaistavissa olevat karttatasot"
             }
         },
+        "AnnouncementsTool": {
+            "buttonLabel": "Valitse Ilmoitukset",
+            "popup": {
+                "title": "Ilmoitukset",
+                "close": "Sulje"
+            },
+        },
```

Comment:
You already have these in the announcement bundle localizations. Why add them again?